pages/design/atomic-template-empty.md

```diff
@@ -50,7 +50,23 @@ _There are no major accessibility support issues known for this rule._
 
 ## Background
 
-- (e.g. WCAG Techniques or links with background information mentioned in Applicability, Expectations or Assumptions)
+### Understanding WCAG 
+
+- [WCAG 2.1 - Understanding 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
```

Comment:
I agree with @carlosapaduarte that it seems weird to have these pre-filled in the template, especially without comments. It kinda look like we *have to* include these links which are just an example of what to put here.

Reply:
I now have added comments stating these are examples and should be removed.